app/content/assignments/index.js

```diff
@@ -105,12 +106,17 @@ const DrillList = () => {
                 paddingBottom: 5,
               }}
             >
-              <Text style={{ fontSize: 20 }}>
-                {drillInfo[assignment.drillId]["drillType"]}
-              </Text>
-              <Text style={{ fontSize: 17, fontStyle: "italic" }}>
-                {drillInfo[assignment.drillId]["subType"]}
-              </Text>
+              <View style={{ flexDirection: "column" }}>
+                <Text style={{ fontSize: 20 }}>
+                  {drillInfo[assignment.drillId]["drillType"]}
+                </Text>
+                <Text style={{ fontSize: 14, fontStyle: "italic" }}>
+                  {drillInfo[assignment.drillId]["subType"]}
+                </Text>
+              </View>
+              {assignment.completed && (
+                <Icon source="check" size={20} color="green" />
```

Comment:
Not sure if we are updating this based on what was said in the discord or not or if it was done on a different branch.